src/mlpack/methods/neighbor_search/knn_main.cpp

```diff
@@ -78,6 +82,9 @@ PARAM_FLAG("single_mode", "If true, single-tree search is used (as opposed to "
     "dual-tree search).", "S");
 PARAM_DOUBLE_IN("epsilon", "If specified, will do approximate nearest neighbor "
     "search with given relative error.", "e", 0);
+PARAM_STRING_IN("effective_error", "If specified, will compare the results "
+    "against the provided distances file, and will print the average relative "
+    "error.", "E", "");
```

Comment:
Sure! Done in: https://github.com/mlpack/mlpack/pull/747/commits/c5fe485b25e93f35b14f2c9fbc6b69ddbe59a837